src/services/jobOwner/talentAPI.ts

```diff
@@ -0,0 +1,71 @@
+import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
+import { IFreelancer } from "components/CandidateCard/CandidateCard";
+import { RootState } from "store/store";
+
+export interface IFilters {
+  keyWords: string;
+  category: string;
+  skills: string[];
+  userId: number;
+}
+
+interface ISaveFreelancer {
+  userId: number;
+  freelancerId: number;
+}
+
+const getTalentApi = createApi({
+  reducerPath: "talent",
+  baseQuery: fetchBaseQuery({
+    baseUrl: process.env.REACT_APP_API_URL,
+    prepareHeaders: (headers, { getState }) => {
+      const { authToken } = (getState() as RootState).user;
+
+      if (authToken) {
+        headers.set("Authorization", `Bearer ${authToken}`);
+      }
+
+      return headers;
+    },
+  }),
+  tagTypes: ["Freelancers"],
+  endpoints: (builder) => ({
+    filter: builder.query<IFreelancer[], IFilters>({
+      query: ({ keyWords, category, skills, userId }) => ({
+        url: `/client/filter/${userId}/${keyWords}/${category}/${skills}`,
```

Comment:
we have file with brakepoint path consts. Let's keep all brakepoints in one place

Reply:
fix it